packages/linter/src/executors/eslint/lint.impl.ts

```diff
@@ -79,7 +79,16 @@ Please see https://nx.dev/guides/eslint for full guidance on how to resolve this
   }
 
   if (lintResults.length === 0) {
-    throw new Error('Invalid lint configuration. Nothing to lint.');
+    for (let i = 0; i < options.lintFilePatterns.length; i++) {
```

Comment:
Shouldn't this be along the lines of...

```ts
await allPatternsIgnored =
  Promise.all(
    options.lintFilesPattern
      .map(async (pattern) => await eslint.isPathIgnored(pattern)
  )
  .every(ignored => ignored)
```

Reply:
In case you have only `.ts` files and the ignore targets those specifically `isPathIgnored` would yield false for `apps/app1/**/*.html` pattern. We want to report all of the ignored patterns, but not all of them have to be ignored.

Running concurrently is a good suggestion.